src/main/webapp/Dialogs/form.js

```diff
@@ -0,0 +1,16 @@
+function showForm() {
+    const formTemplate = document.querySelector("#demo-form");
```

Comment:
not fussed which one is used but should probably not use querySelector here for an ID and 2 lines down use getElementById, looks strange to do the same thing in two different ways
```suggestion
    const formTemplate = document.getElementById("#demo-form");
```